osu.Desktop/Windows/GameplayWinKeyBlocker.cs

```diff
@@ -5,32 +5,32 @@
 using osu.Framework.Bindables;
 using osu.Framework.Graphics;
 using osu.Framework.Platform;
+using osu.Game;
 using osu.Game.Configuration;
 
 namespace osu.Desktop.Windows
 {
     public class GameplayWinKeyBlocker : Component
     {
-        private Bindable<bool> allowScreenSuspension;
         private Bindable<bool> disableWinKey;
+        private Bindable<bool> localUserPlaying;
 
-        private GameHost host;
+        [Resolved]
+        private GameHost host { get; set; }
 
-        [BackgroundDependencyLoader]
-        private void load(GameHost host, OsuConfigManager config)
+        [BackgroundDependencyLoader(true)]
+        private void load(OsuGame game, OsuConfigManager config)
         {
-            this.host = host;
-
-            allowScreenSuspension = host.AllowScreenSuspension.GetBoundCopy();
-            allowScreenSuspension.BindValueChanged(_ => updateBlocking());
+            localUserPlaying = game.LocalUserPlaying.GetBoundCopy();
```

Comment:
`game` nullable?

Reply:
Ah yeah, realised halfway that it can never be null (this component is only ever used in `OsuGameDesktop`). Will remove the null allowance.